README.md

```diff
@@ -202,6 +202,42 @@ BACKFILL_CACHE_PROVIDER="npm"
 BACKFILL_CACHE_PROVIDER_OPTIONS='{"npmPackageName":"...","registryUrl":"..."}'
 ```
 
+### Skipping cache locally
+
+Sometimes in a local build environment, it is useful to compare hashes to
+determine whether to execute the task without having to explicitly use a
+separate directory for the cache. While running an incremental task, he
+generated build artifacts are the caches themselves.
```

Comment:
I don't understand this analogy. Maybe the first sentence in this paragraph is enough?

Reply:
okay, removed the sentence.